src/collective/cover/config.py

```diff
@@ -37,3 +37,5 @@
 # In case that no value was set to that field
 # we need to detect it and use it
 DEFAULT_SEQUENCEWIDGET_VALUE = '--NOVALUE--'
+
+ANNOTATION_PREFIXES = ('plone.tiles.data', 'plone.tiles.configuration', 'plone.tiles.permission')
```

Comment:
these constants are defined somewhere else:

* https://github.com/collective/collective.cover/blob/1.6b4/src/collective/cover/tiles/configuration.py#L17
* https://github.com/collective/collective.cover/blob/1.6b4/src/collective/cover/tiles/permissions.py#L9
* https://github.com/plone/plone.tiles/blob/2.0.0b3/plone/tiles/data.py#L37

Reply:
did you saw this?

Reply:
Yes, I saw, need to review this